felix/bpf/proxy/syncer.go

```diff
@@ -720,8 +724,12 @@ func (s *Syncer) updateService(skey svcKey, sinfo k8sp.ServicePort, id uint32, e
 		if ep.GetIsLocal() {
 			continue
 		}
-		if err := s.writeSvcBackend(id, uint32(cnt), ep); err != nil {
-			return 0, 0, err
+
+		// eps could contain Ready and Terminating pods but only write Ready pods to backend.
+		if ep.IsReady() {
```

Comment:
Update code to prevent terminating pods from being added to the bpf maps as outlined above @tomastigera